src/_posts/languages/python/django/2000-01-01-start.md

```diff
@@ -233,6 +233,29 @@ Then change the environment variable of your application with a coma-separated l
 scalingo -a app-name env-set ALLOWED_HOSTS=app-name.osc-fr1.scalingo.io,example.com
 ```
 
+### Automatic database migrations
+
+If your Django application is related to a database, you can set it up to deploy automatically after a deployment, 
+thanks to the [postdeploy hook]({% post_url platform/app/postdeploy-hook %}).
```

Comment:
I don't understand what "deploying automatically" a database means. You may have used the wrong word?

Reply:
This is totally bad worded, I will check that